starknet-accounts/src/single_owner.rs

```diff
@@ -116,6 +93,21 @@ where
         }
         execute_calldata.push(nonce); // nonce
 
+        let transaction_hash = compute_hash_on_elements(&[
+            PREFIX_INVOKE,
+            FieldElement::ZERO, // version
+            self.address,
+            get_selector_from_name("__execute__").unwrap(),
```

Comment:
Wouldn't it make sense to have this as a const as well? Could be reused just below, on line 113.

Reply:
Makes sense. Even better, I would like to make these util functions themselves const too. Will look into that later.